code/modules/fields/timestop.dm

```diff
@@ -89,6 +92,7 @@
 
 /datum/proximity_monitor/advanced/timestop/proc/unfreeze_throw(atom/movable/AM)
 	var/datum/thrownthing/T = frozen_throws[AM]
+	escape_the_negative_zone(AM)

```

Comment:
Why in unfreeze_throw?

Reply:
I honestly have no clue what this proc is for since comments are strictly forbidden in tgstation.

Simple answer: everything that gets coloured in freeze_atom needs to be uncoloured but unlike when atoms are frozen, there is no communal code guaranteed to be executed (players can be pulled out of the edge of timestop early by external forces) so I bunged it in all the unfreezes to also make it easier to change other code.

Side question: do thrown objects not get coloured because I should be manipulating T in freeze_throwing and then going to uncolour it in unfreeze_throwing or is the lack of comments really hampering my ability to understand what the hell is going on?

Reply:
I will remove the line since I can't even colour thrown objects.